ipod/utils.py

```diff
@@ -273,3 +274,70 @@ def identify_found_missed_and_new(
         obs_ids_prev.filter(missed_mask).to_pylist(),
         obs_ids_iter.filter(new_mask).to_pylist(),
     )
+
+
+def assign_duplicate_observations(
+    orbits, orbit_members: FittedOrbitMembers
+) -> Tuple[FittedOrbits, FittedOrbitMembers]:
+    """
+    Assigns observations that have been assigned to multiple orbits to the orbit with t
+    he most observations, longest arc length, and lowest reduced chi2.
+    Parameters
+    ----------
+    orbit_members : `~thor.orbit_determination.FittedOrbitMembers`
+        Fitted orbit members.
+    Returns
+    -------
+    filtered : `~thor.orbit_determination.FittedOrbits`
+        Fitted orbits with duplicate assignments removed.
+    filtered_orbit_members : `~thor.orbit_determination.FittedOrbitMembers`
+        Fitted orbit members with duplicate assignments removed.
+    """
+    # Sort by number of observations, arc length, and reduced chi2
+    # Here we assume that orbits that generally have more observations, longer arc lengths,
+    # and lower reduced chi2 are better as candidates for assigning detections
+    # that have been assigned to multiple orbits.
+    sorted = orbits.sort_by(
```

Comment:
`sorted` is the name of a python built-in.

Reply:
Renamed!